apps/video/js/video.js

```diff
@@ -1289,6 +1325,22 @@ function handleSliderTouchMove(event) {
   dom.player.fastSeek(dom.player.duration * pos);
 }
 
+function handleSliderKeypress(event) {
+  // The standard accessible control for sliders is arrow up/down keys.
+  // Our screenreader synthesizes those events on swipe up/down gestures.
+  // Currently, we only allow screen reader users to adjust sliders with a
+  // constant step size (there is no page up/down equivalent). In the case
+  // of videos, we make sure that the maximum amount of steps for the entire
+  // duration is 20, or 2 second increments if the duration is less then 40
+  // seconds.
+  var step = Math.max(dom.player.duration / 20, 2);
+  if (event.keyCode === event.DOM_VK_DOWN) {
+    dom.player.currentTime -= step;
```

Comment:
This is one example of the problems of having two video player implementations (one in the video app and one in shared/js/media used by gallery and camera). We are working on fixing this by creating a video player web component. In any event, the video app code is more active and current. There was an explicit decision to use fastSeek as opposed to manipulating currentTime. That this change was only made in the video app and not in the video player code used by gallery and camera was an oversight that was not fixed due to our goal to have a single web component shared by all apps using a video player.